src/test/java/redis/clients/jedis/tests/commands/ListCommandsTest.java

```diff
@@ -328,6 +329,86 @@ public void lpop() {
 
   }
 
+  @Test
+  public void lpos() {
```

Comment:
Test empty LPosParams.

Also, move this test method after all List related tests.

Reply:
Refactored test towards the done and added a assertion with an empty paras list.